snf/inject.go

```diff
@@ -148,8 +148,11 @@ type Sender struct {
 	frags []C.struct_snf_pkt_fragment
 
 	// buffers for injecting in bulk
-	pkts []byte
+	pkts []uintptr
 	len  []uint32
+
+	// protect the memory from GC
+	guardPkts [][]byte
```

Comment:
just a note, this is valid if Sender is allocated in heap.

Reply:
Added the note in documentation